app/ports/management/commands/autoload-initial-data.py

```diff
@@ -0,0 +1,34 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from ports.models import Port, LastPortIndexUpdate, BuildHistory, Builder
+
+
+class Command(BaseCommand):
+
+    help = "Populates the database with Initial data after fetching portindex.json from rsync"
+
+    def handle(self, *args, **options):
+        if Port.objects.count() or Builder.objects.count() > 0:
+            raise CommandError("The database is not empty. Command cannot run.")
+
+        # Fetch from rsync
+        Port.RsyncHandler().sync()
+
+        # Open the file
+        data = Port.RsyncHandler().open_file()
```

Comment:
In a way it could make sense to run `sync_and_open_file()` (just a single command doing both), but it's also ok as it is.